.circleci/config.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.6.6-jessie-node
+
+    steps:
+      - checkout
+      - run:
+          name: Install Python deps in a venv
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            pip install django==2.0
```

Comment:
Aren't requirements handled in the `setup.py`?

https://github.com/Work4Labs/django-jet/blob/master/setup.py

Reply:
Yes they are but `python setup.py install` is building the full django-jet dashboard which is not needed to run tests.

Also I just saw that tests are good with django 2.0 but fail with dajngo 2.1
I have absolutely no idea why

Reply:
Django 2.1 introduces a bunch of breaking changes – I'm encountering issues with AppRegistry and mongo engine right now. Will link to my commit here, could be related. Otherwise, as Jet does a bunch of low level admin tweaks it's always a challenge to work around new Django versions.

Do note that we have no plans currently to upgrade `work4us` to Django 2.1, so I wouldn't spend too much time on ensuring that is working at this time.

Reply:
Ok, so is this good for you ?

Reply:
For now I'd test with Django 1.10 as that's what `work4us` uses for now, if you're only going to be testing a single Django version.

Reply:
I don't have access to this repo 😄